README.md

```diff
@@ -1,28 +1,279 @@
-# XiSearch
-Actually a library containing several tools - most prominently a cross-link MS search engine.
+## xiSEARCH 1.7.6.7 documentation
 
-XiSearch is implemented as a Java Application. Therefore it requires that java is installed. We recommend to use the the latest update of JAVA version 8.
 
+XiSearch is implemented as a Java Application. Therefore it requires that java is installed. We recommend to use the the latest update of JAVA version 8 or above. 
+The latest version as binary can be downloaded from https://rappsilberlab.org/downloads/
 
-The latest version as binary can be downloaded from
-https://rappsilberlab.org/downloads/
+### Background
+xiSEARCH is a search engine for crosslinking mass spectrometry (crosslinking MS). It supports data acquired with ThermoFisher Orbitrap instruments (.raw format) that have been converted to peak. files (.mgf format), for example with [ProteoWizard MsConvert](https://proteowizard.sourceforge.io/). It then searches the peakfiles against a sequence database in .fasta format to identify crosslinked peptide pairs from mass spectra. The search algorithm uses a target-decoy approach outlined in XXX, which enables false-discovery rate (FDR) estimation. The FDR calculation on the xiSEARCH result is performed by xiFDR.
```

Comment:
```suggestion
xiSEARCH is a search engine for crosslinking mass spectrometry (crosslinking MS). It supports data acquired with ThermoFisher Orbitrap instruments (.raw format) that have been converted to peak files (.mgf format), for example with [ProteoWizard MsConvert](https://proteowizard.sourceforge.io/). It then searches the peakfiles against a sequence database in .fasta format to identify crosslinked peptide pairs from mass spectra. The search algorithm uses a target-decoy approach outlined in XXX, which enables false-discovery rate (FDR) estimation. The FDR calculation on the xiSEARCH result is performed by xiFDR.
```

Do you want a reference for target-decoy in general or something that states how it actually works in xi?

Reply:
your call- i would say your 2 publications on T-D?

Reply:
added my two FDR papers as reference

Reply:
why no more link to rappsilber lab?

Reply:
ah github confused me